test/operations/wait.go

```diff
@@ -45,10 +45,14 @@ func networkUpCondition() (bool, error) {
 }
 
 func proverUpCondition() (bool, error) {
-	return ops.ProverUpCondition()
+	// return ops.ProverUpCondition()
+	// TODO: remove time sleep (mock prover port is not matched in the condition check)
+	const t time.Duration = 5
+	time.Sleep(t * time.Second)
+	return true, nil
```

Comment:
Why do you introduce a sleep instead of having the previous mechaninsm?

Reply:
here we are using the mock prover port with 50052, but the actual port is 50051 and this is used in `condition` function, of course it is defined in node repo. We can't modify them.

Reply:
we can do a port forwarding `50051:50052` to the `docker-compose`, because the test is going to access the prover via the host network